apps/web/next.config.js

```diff
@@ -25,44 +26,68 @@ const nextConfig = {
 			'apistage.gauzy.co',
 			'gauzy.s3.wasabisys.com'
 		]
+	}, // Optional build-time configuration options
+	sentry: {
+		// See the sections below for information on the following options:
```

Comment:
@NdekoCode very weird that tons of those lines are commented out, but we have same lines below in this file! E.g. tunnelRoute in https://github.com/ever-co/ever-teams/pull/1771/files#diff-a5de0fa0d80f14a44e9bdf2ed5d0cbb3a7bd5fdcacfbc1d03b5dcb57fc4943c1R86

Reply:
Understanding the sentry configuration was difficult for me, there are comments that come with the initial configuration but there are others that I added at the time of testing.
I have these comments to help me and those who will want to understand the updates, there are also several configuration concerning the build of the application like the options:
// disableServerWebpackPlugin: true,
// disableClientWebpackPlugin: true,
  which you need to uncomment if you encounter a permission problem during build, for example, if you don't have a SENTRY_AUTH_TOKEN.